src/librustc_resolve/lib.rs

```diff
@@ -1110,6 +1140,11 @@ pub struct Resolver<'a, 'tcx:'a> {
 
     used_imports: HashSet<(NodeId, Namespace)>,
     used_crates: HashSet<CrateNum>,
+
+    // Callback function for intercepting walks
```

Comment:
I think some more docs are in order. For example, what is the purpose of this boolean flag?